feat(content): Donna & Charles Follow Up Mission (#10665)
endless-sky/endless-sky: data/human/human missions.txt

	minor
	source "Earth"
	destination "Shangri-La"
	to offer
		has "event: charles donna timer"
		has "outfit: Luxury Accommodations"
	passengers 3
	on offer
		conversation
			`As you enter the spaceport, you see a familiar sight. The elderly couple Charles and Donna Singer, who you helped move to Earth a while ago, are talking to a young well-dressed man. Donna looks much the same as you remember her, but Charles is sporting a bandage on his head, and one of his arms looks like it's in a sling. When they see you, Donna points excitedly in your direction, and the entire group moves to meet you.`
			`	"Captain! I never thought we would see you again," Donna begins. "I'd like to introduce you to my son, Abel." She gestures to the young man, who shakes your hand vigorously. Now that you're close, you can see that he is dressed very finely. His suit is made of expensive wool with ebony buttons, his shoes are emblazoned with designer logos, and you can see a jewel-encrusted watch peeking out from underneath the well-tailored cuff of one arm.`
			`	"It's great to meet you, Captain...?"`
			choice
				`	"<first> <last>."`
				`	"<last>."`

> 			`	"Captain <last>," he smiles. "How very nice to meet you indeed. My parents here told me what you did for them. If I may introduce myself more fully, I am Abel Singer, Junior Vice President of Marketing Operations for Syndicated Systems. I want to thank you for going out of your way to help my parents when they needed it most."`

Author: ~~can't wait for you to add this guy to the Syndicate Business missions~~

			choice
				`	"It was nothing."`
				`	"What happened to Charles?"`
					goto injuries
				`	"How could you leave them to rot...?"`
					goto rot

> 			`	"It was not nothing," Donna blurts out. "You were there for us when nobody else cared. You didn't even accept the agreed upon payment! Our former employers, our friends, even our children..." She trails off.`

Reviewer: The player could have taken the payment in the last mission, so I think that line should be removed or you need to add a condition to the last mission & a branch here.

Author: This won't be offered if the player took the payment.

Reviewer: Ah, I see. My bad, this works as is -- feel free to resolve

			label rot
			`	"I've not been the best son," Abel says, staring down at his feet. "My parents are very dear to me, of course. But I was so busy with work. I thought they could take care of themselves. So I left them to their own devices. I didn't even think to wonder about their financial position, and my father here was too proud to say it out loud." He turns to look his father in the eyes. "Still, I should have known better. I'm sorry, dad."`
			label injuries

> 			`	Charles grunts, but it is his wife who speaks up next. "Shortly after we arrived, a gang of local teenagers began terrorizing our retirement community. It was mostly just vandalism, some petty theft, and even some burglaries. Nothing would have happened if Charles had just given up the necklace-"`

Reviewer 2: "It was mostly... even" doesn't work here. This might flow better.

```suggestion
			`	Charles grunts, but it is his wife who speaks up next. "Shortly after we arrived, a gang of local teenagers began terrorizing our retirement community. At first it was just vandalism, but eventually it became petty theft, burglary... Nothing would have happened if Charles had just given up the necklace-"`
```

			`	"Give up your necklace?!" Charles snaps. "You know I couldn't do that. It was your mother's - the gift she gave you on the day of our wedding, to show you how proud she was. I'd face down those bullies a hundred times if it meant saving it."`
			`	A silence follows, but it's not uncomfortable. Donna looks at Charles with tenderness; Charles looks at her with love. Abel looks at both of them with sorrow and regret. Finally, the three of them turn to you.`
			`	"I am taking my parents to come live with me on <planet>. I think it'd be fitting if you took us. This time, I will spare no expense to ensure their safety and comfort. We will pay you <payment> for the trip."`
			choice

> 				`	"Welcome aboard, Singer family."`

Reviewer 2: I can't see any universe in which I would say this... is there a way to phrase it so it doesn't sound like I'm some sort of game show host?

Author: I'm not sure. This was the original wording... what about this:
```suggestion
				`	"You're welcome on my ship."`
```

					accept
				`	"Sorry, I can't. I've got other things to do."`
			`	"Well, the best of luck to you, then," Abel immediately replies. "We wish you well!" adds Charles. They all go in search of another captain.`
				decline
	on visit
		dialog `You land on <planet>, but realize that your escort carrying the Singer family hasn't entered the system yet. Better depart and wait for it to arrive.`
	on complete
		payment 85000
		conversation

> 			`When you land on <planet>, Abel insists on carrying all of the family's personal effects. Donna and Charles are almost reluctant to say goodbye to the Luxury Accommodations on your ship, but they look ahead smiling as their son leads them on to the better, happier retirement that they have well earned.`

Reviewer 2: While outfits are usually capitalized, luxury accommodations are vague enough that they tend to not be, and I think having them lowercase fits better with the sentence:
```suggestion
			`When you land on <planet>, Abel insists on carrying all of the family's personal effects. Donna and Charles are almost reluctant to say goodbye to the luxury accommodations on your ship, but they look ahead smiling as their son leads them on to the better, happier retirement that they have well earned.`
```
